.github/workflows/release.yml

```diff
@@ -6,16 +6,43 @@ on:
       version:
         description: 'The version that is going to be released. Should be in format 7.y.z'
         required: true
-        default: '7.y.z'
+        default: ''
+      forceRecreateTags:
```

Comment:
I thought @themr0c said we would never need to retag or rebranch docs, since docs only live in master branch.

Reply:
Can you clarify what this setting does?

For a `7.__<something>__.0`
release, creating a new `7.__<something>__.x` branch is fine and desired.

For subsequent releases (`7.__<something>__.1`, `7.__<something>__.2`, ...):

* We are actively maintaining the  `7.__<something>__.x` branches corresponding to downstream releases. They include a lot of backportes and fixes, we can't afford any overwrite.

* Recreating / overwriting a `7.__<something>__.x` branch that already exist is a no-go.

* Updating the `antora-playbook.yml` is desired. (Branching from `7.__<something>__.x` + PR to `7.__<something>__.x`)
We (as docs team) are not using tags at all at the moment, but I understand they can make sense for historical or forensics purpose, to retrieve the status of the docs at the day of a release.

Reply:
it is just to recreate tags, should release have to be rerun several times for some reason, caused by some unexpected error. Bugfix branch will not be recreated